### DISCUSSION TITLE
How to create table with multiline cells?

### DISCUSSION THREAD

**Discussion by R0land013** [py-pdf/fpdf2#550] [Q&A]
I want to know how to create a table that can support a cell with a long text, so the text is wrapped, and all cells in that row resize to the same height.
I wanted to know if there is a simple way to do this. Because I can imagine a way to do it, but more code and logic is needed 😄 .

**Comment by Lucas-C** [comment]
Hi @R0land013!

Do you want to build a HTML `<table>`? Sadly this is not currently supported, check #91

If you are not using `.write_html()`, have you checked this documentation page?
https://pyfpdf.github.io/fpdf2/Tables.html

**Reply by R0land013** [reply]
Hi.
Yes, I wanted it. Finally I found the [xhtml2pdf](https://pypi.org/project/xhtml2pdf/), this package allows me to do what I have described. Thank you very much for your time.
This discussion can be closed.

**Comment by gmischler** [comment]
Define "simple"... :wink: 
The general strategy for doing that is outlined in [this answer](https://github.com/PyFPDF/fpdf2/issues/494#issuecomment-1219196208) to a similar question.
